Makefile

```diff
@@ -321,6 +322,8 @@ export CSV="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
 .PHONY: bundle-ocp
 bundle-ocp: yq bundle-base ## Generate bundle manifests and metadata for OCP, then validate generated files.
 	$(KUSTOMIZE) build config/manifests/ocp | $(OPERATOR_SDK) generate --verbose bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	# use custom build.Dockerfile
```

Comment:
what do we need this for?

Reply:
it also fails CI bundle image build

Reply:
the `bundle.Dockerfile` is rebuilt by `$(OPERATOR_SDK) generate --verbose bundle ...` so no matter what I commit, it is overwritten and `verify-diff` fails

> it also fails CI bundle image build

I think it is because I don't copy `bundle.ci.Dockerfile` in the container before `RUN make-ocp`